product/pom.xml

```diff
@@ -10,19 +10,15 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-	<maven.compiler.source>11</maven.compiler.source>
-	<maven.compiler.target>11</maven.compiler.target>
+	<maven.compiler.source>8</maven.compiler.source>
+	<maven.compiler.target>8</maven.compiler.target>
         <aio.lib.path>${basedir}/lib</aio.lib.path>
         <jvm.args>-Xms1024m -Xmx1024m -Djava.library.path=${aio.lib.path}</jvm.args>
         <version.EAP5>4.6.1.GA_CP12</version.EAP5>
         <version.EAP6>4.16.4.Final </version.EAP6>
         <version.EAP6.latest>5.0.4.Final-SNAPSHOT</version.EAP6.latest>
         <version.EAP7-IDLJ>5.0.3.Final</version.EAP7-IDLJ>
-
-        <version.jacorb.patched>2.3.1.patched</version.jacorb.patched>
-        <version.jacorb.EAP5>4.6.1.GA</version.jacorb.EAP5>
-        <version.jacorb.EAP6>2.3.1.jbossorg-1</version.jacorb.EAP6>
-        <version.jacorb.EAP6.latest>2.3.1.jbossorg-1</version.jacorb.EAP6.latest>
+        <version.org.jboss.jboss-transaction-spi>7.5.2.Final</version.org.jboss.jboss-transaction-spi>
```

Comment:
This version is quite old now, the latest is 7.6.1.Final (and there is also 7.6.0.Final).
The version in `narayana/pom.xml` also needs updating and the version in `ArjunaJTA/jta/pom.xml ` needs to reference the property `version.org.jboss.jboss-transaction-spi`.